rxandroid/src/main/java/io/reactivex/android/schedulers/AndroidSchedulers.java

```diff
@@ -47,6 +48,28 @@ public static Scheduler from(Looper looper) {
         return new HandlerScheduler(new Handler(looper));
     }
 
+    @TargetApi(Build.VERSION_CODES.P)
```

Comment:
This shouldn't be needed

Reply:
I agree! But lint fails without it despite the if-check :|

Reply:
Did you try just using `28` as a literal? I prefer avoiding the codenames (which are thankfully almost dead) and letters (which we should be trying to kill next).

Reply:
Huh, yeah using jus 28 worked. c9e2bbf

RIP dessert club